anyway/utilities.py

```diff
@@ -195,3 +198,36 @@ def chunks(l, n):
     """Yield successive n-sized chunks from l."""
     for i in range(0, len(l), n):
         yield l[i : i + n]
+
+
+def is_valid_number(phone: str) -> bool:
+    try:
+        phone_obj = phonenumbers.parse(phone, "IL")
+        return phonenumbers.is_valid_number(phone_obj)
+    except NumberParseException:
+        return False
+
+
+def check_is_a_safe_redirect_url(url: str) -> bool:
+    url_obj = urlparse(url)
+    if url_obj.scheme not in ["https", "http"]:
```

Comment:
Why allow http?

Reply:
In case of local debugging - `http://127.0.0.1:5000/`,
I can minimize it to localhost only.